File: app/src/main/java/com/example/bakerystore/models/CartItemResponse.kt
package com.example.bakerystore.models

data class CartItemResponse(
    val cartItemId: Int,
    val userId: Int,
    val productId: Int,
    val productName: String,
    val description: String?,
    val imageUrl: String?,
    val price: Double,
    val quantity: Int,
    val total: Double
)

File: backend/BakeryStoreApi/DTOs/CartItemResponse.cs
namespace BakeryStoreApi.DTOs
{
    public class CartItemResponse
    {
        public int CartItemId { get; set; }

        public int UserId { get; set; }

        public int ProductId { get; set; }

        public string ProductName { get; set; } = string.Empty;

        public string? Description { get; set; }

        public string? ImageUrl { get; set; }

        public decimal Price { get; set; }

        public int Quantity { get; set; }

        public decimal Total { get; set; }
    }
}